documentation/index.rst

```diff
@@ -31,6 +31,16 @@ You can check for pre-releases by passing ``--pre`` to the ``pip install`` comma
 
 Installing pre-releases gives you early access to features or bug fixes, and helps with testing.
 
+Quick Start
+===========
+
+After installing ``mots``, follow these instructions to make changes to module information:
+
+- Update mots.yaml as needed (for example, by adding a new person, modifying information, deleting old modules, etc.)
+- Run ``mots export-and-clean``
+- Commit changes and push
+
+See additional sections below for more information on :ref:`adding a module<Adding a Module>`, :ref:`adding a submodule<Adding a Submodule>`, :ref:`adding a new person<Adding a new Person>`, :ref:`cleaning<Cleaning>`, and :ref:`exporting<Exporting>`.
```

Comment:
`mots init` should probably be mentioned, no?

Reply:
These are the most common operations. `init` is rarely used and likely doesn't need to be here, but can add it anyway (why not).